src/components/organisms/NotFound.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Pane, Text, Heading } from 'evergreen-ui';
+
+const NotFound: React.FC = () => {
+    return (
+        <Pane display="flex" height="100%" justifyContent="center" alignItems="center" flexDirection="column">
+            <Heading>404</Heading>
+            <Text>Oops! Her ser det ut som det ble noe krøll.. Vi finner ikke siden du leter etter.</Text>
```

Comment:
Ellipsis' are usually written with [three](https://en.wikipedia.org/wiki/Ellipsis) dots.

Reply:
They are not actually three dots, it's its own unicode character.

…

You can copy the symbol above.

Reply:
Actually...

Reply:
Refer to the link in my previous comment @FredrikAugust